crates/gossip/tests/historical_load.rs

```diff
@@ -95,12 +117,10 @@ async fn historical_load() {
         "Completed rounds with agent 2 should have a value"
     );
 
-    let required_rounds =
-        total_size / K2GossipConfig::default().max_gossip_op_bytes as usize + 1;
     assert!(
-        completed_rounds_with_agent_2.unwrap() <= required_rounds as u32,
```

Comment:
There is a behaviour change here and this is where I should have spent more time on this test in the first place. The disc sync has to complete before the disc top hash matches and the test will move on. That means it's not just a matter of a round or two to discover all the op ids. I'm see the op ids discovered quickly but then we have to fetch all the data (about 40Mb) before we can proceed to syncing the rings. So we can place an upper bound on this but not really an assertion because it depends on compute speed. Locally I'm seeing it complete ~7 rounds. The first couple sync disc ops and then it's waiting on fetch.